draft-ietf-rtcweb-jsep.xml

```diff
@@ -1498,11 +1536,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               Section 2.1, containing the dummy value "9 IN IP4 0.0.0.0", because
               no candidates have yet been gathered.</t>
 
-              <t>An "a=msid" line, as specified in <xref
-              target="I-D.ietf-mmusic-msid"></xref>, Section 2.</t>
-
-              <t>An "a=sendrecv" line, as specified in <xref
-              target="RFC3264"></xref>, Section 5.1.</t>
+              <t>If the RtpTransceiver's RtpSender is active, an "a=msid" line,
```

Comment:
This seems like it should be whether the sender has a MST, not whether it is active.